packages/docs/types/configTypes.ts

```diff
@@ -15,18 +16,26 @@ export enum BlockType {
 
 export type ApiDocsBlock =
   | {
-  type: BlockType.TITLE | BlockType.SUBTITLE | BlockType.PARAGRAPH | BlockType.HEADLINE,
-  translationString: TranslationString,
-}
+      type:
+        | BlockType.TITLE
+        | BlockType.SUBTITLE
+        | BlockType.PARAGRAPH
+        | BlockType.HEADLINE,
+      translationString: TranslationString,
+      text: string,
```

Comment:
This is incorrect. There should be no text.
In any of examples.

Reply:
@asvae How about this:
![image](https://user-images.githubusercontent.com/24375400/80725455-c557fb00-8b0b-11ea-80a3-1d888b1e74f7.png)

Reply:
Translation string should be fine here. It's used for anchors I suppose?